pyproject.toml

```
@@ -30,7 +30,7 @@ Repository = "https://github.com/NiklasRosenstein/python-github-bot-api"
 [tool.poetry.dependencies]
 python = "^3.8"
 "nr.functional" = "^0.2.0"
-cryptography = "^39.0.2"
+cryptography = "*"
```

```suggestion
cryptography = ">=39.0.2"
```

Should we do this to lower bound at least on a known old working version of the package?

---

The latest version without any vulnerabilities is 42.0.5 shall we use the below perhaps? 

`"^42.0.5"`